File: 3st/Seojun/n보다 커질 때까지 더하기.java
class Solution {
    public int solution(int[] numbers, int n) {
        int sum = 0;
        for (int i = 0; i < numbers.length; i++) {
            sum += numbers[i];
            if (sum > n) {
                return sum;
            }
        }
        return sum;  
    }
}

File: 3st/Jaeyoung/n보다 커질 때까지 더하기.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

// numbers_len은 배열 numbers의 길이입니다.
int solution(int numbers[], size_t numbers_len, int n) {
    int answer = 0;
    int sum = 0;
    while ( answer <= n ) {
        answer += numbers[sum++];
    }
    return answer;
}